hummingbot/core/utils/eth_gas_station_lookup.py

```diff
@@ -72,6 +120,23 @@ def refresh_time(self):
     def gas_price(self):
         return self._gas_prices[self.gas_level]
 
+    @property
+    def gas_limit(self):
```

Comment:
if this is a dictionary, might be better to name it with plural, e.g. gas_limits